src/components/SearchList.js

```diff
@@ -1,33 +1,112 @@
 // 3rd party libraries - core
 import React from 'react';
-import {View, Text, StyleSheet} from 'react-native';
-import {Card} from 'react-native-elements';
+import {View, Text, StyleSheet, ScrollView, ImageBackground} from 'react-native';
+import {Card, ListItem, Button, Tile, List, Image, Badge, Icon} from 'react-native-elements';
 // 3rd party libraries - additional
-
 // styles and language
-
-// our components - core
+import { badgeGreen, limeGreenTitle} from '../styles/index';
 // our components - additional
 
-const SearchList = ({sites}) => {
 
+const SearchList = ({sites}) => {
+  const{textStyle, IconContainer, subtitleView,siteAvatarStyle, siteAvatarContainerStyle, titleView, nearestTownStyle, badgeWrapperStyle, badgeContainerStyle} = styles
+  if(!sites){
+    return( <Text> No Sites Available </Text>)
+  }else{
     return (
-        <Card>
-            <Text>
-                List Results of Sites
-            </Text>
-        </Card>
-    );
+  <ScrollView>
+     <List>
+       {
+         sites.map(site => {
```

Comment:
If you're cool with it, let's use `lodash` for these ES6 functions. It gives us more practice with that library, and many times their implementation is more performant than the native language ones.